### DISCUSSION TITLE
希望可以加入将结果复制成markdown格式

### DISCUSSION THREAD

**Discussion by RyanZeDev** [MuiseDestiny/zotero-gpt#193] [Ideas]
想着不仅看文献，还可以把一些英文的手册或者其他的弄到zotero里看，用gpt做markdown笔记

**Comment by MuiseDestiny** [comment]
双击就会复制gpt的原本回答（插件的输出框是md渲染后的形式），若规定回答md双击复制就会达到你说的效果。

**Reply by RyanZeDev** [reply]
索嘎，阿里嘎多美羊羊桑
